### PR TITLE
Create db queries

### PR DESCRIPTION
**Pull request by elizabethcan**
In this branch I added a few DB queries, the most important of which is `getAllReviewsForStore` which I will use to render the reviews on the item page. I will most likely need to add another query to get the storeId for the item that we are currently viewing so that I can hit this endpoint properly. This branch doesn't change anything on the front end except that my container now has a `reviews` key on its state which will hold the reviews that the page needs to display.

### CODE REVIEW

File: client/src/components/Container.jsx [hunk 1]
```diff
@@ -9,16 +10,31 @@ class Container extends React.Component {
...
+  }
+
   toggleModal() {
     this.setState({
       showModal: !this.state.showModal
     });
   }
+
+  getReviewInfo(storeId) {
+    axios.get(`/stores/${storeId}/reviews`)
```
Comment (jasonto510) [inline client/src/components/Container.jsx:31]: Is axios easier to use than jquery?

File: server/index.js [hunk 2]
```diff
@@ -13,9 +13,32 @@ app.get('/items/:itemId/reviews', (req, res) => {
...
-      var parsedData = JSON.parse(data)
       console.log('success getting reviews for item');
-      res.status(200).send(parsedData);
+      res.status(200).send(data);
+    }
+  });
+});
+
+app.get('/reviews/:reviewId', (req, res) => {
+  db.getInfoForReview(1, (err, data) => {
```
Comment (jasonto510) [inline server/index.js:23]: Is this going to be 1 or the id sent from the client?

**Review by jasonto510 — APPROVED** [review]
Everything looks fine, I'm just wondering if there's a better way to select everything you want from the database

File: client/src/components/Container.jsx (hunk 1 above)
Comment (elizabethcan) [reply client/src/components/Container.jsx:31]: I don't have too much of an issue with jquery but I thought I'd give axios a try. It was pretty easy!

File: server/index.js (hunk 2 above)
Comment (elizabethcan) [reply server/index.js:23]: this will be the id from the client

Comment (elizabethcan) [reply server/index.js:23]: > Everything looks fine, I'm just wondering if there's a better way to select everything you want from the database

what would you suggest?
I'm going to merge this so I can move forward, but if you have a suggestion I'd be happy to hear about it and possibly refactor.

Comment (elizabethcan) [reply server/index.js:23]: I don't know why that got commented on this thread specifically

**Pull request merged**